rust_snuba/src/metrics/global_tags.rs

```diff
@@ -0,0 +1,74 @@
+use std::collections::BTreeMap;
```

Comment:
Mind adding some unit tests for this module?

Reply:
ok i copied the tests from statsdproxy's AddTags middleware here.